examples/react-apollo/nhost/nhost.toml

```diff
@@ -28,7 +28,7 @@ httpPoolSize = 100
 version = 18
 
 [auth]
-version = '0.21.4'
+version = 'local'
```

Comment:
leftover?

Reply:
thx - I've just updated it to the latest auth version